docs/docs/cmd/planner/task/task-reference-remove.md

```diff
@@ -0,0 +1,39 @@
+# planner task reference remove
+
+Removes the reference from the Planner task.
+
+## Usage
+
+```sh
+m365 planner task reference remove [options]
+```
+
+## Options
+
+`-u, --url [url]`
+: URL location of the reference.  Specify either `alias` or `url` but not both.
```

Comment:
`reference.  Specify eithe..` lets remove the double space here. And also in the option below